### DISCUSSION TITLE
Switching Databases

### DISCUSSION THREAD

**Discussion by GatTF** [dalenguyen/firebase-wordpress-plugin#197] [Q&A] [answered]
How do I switch my storage of new Wordpress users from rtdb to Firestore, using the Integrate Firebase PRO plugin so that it will dynamically save Users to my Firestore db and not rtdb?

Currently this has been set up by another developer (who I have no contact with) to save to rtdb and it works; however, I would like to change that to the Firestore db (within the same Firebase project).

**Comment by dalenguyen** [comment, marked as answer]
Hi @GatTF, you can go to Firebase > WordPress tab and switch the database type to Firestore. 

<img width="729" alt="image" src="https://user-images.githubusercontent.com/14116156/170723149-7c676817-333c-41e3-9059-52958f684397.png">

It will only affect new users. Existing users will be in RTDB, and you need to migrate them to Firestore if you want to.

If you need further assistance, you can send an email to dale@dalenguyen.me
